framework/source/class/qx/core/Property.js

```diff
@@ -891,6 +894,48 @@ qx.Bootstrap.define("qx.core.Property",
 
 
     /**
+     * Emit code for the equality check evaluation
+     *
+     * @param code {String[]} String array to append the code to
+     * @param clazz {Class} the class which originally defined the property
+     * @param config {Object} The property configuration map
+     * @param name {String} name of the property
+     */
+    __emitIsEqualFunction : function (code, clazz, config, name)
+    {
+      code.push('var equ=');
+
+      if (typeof config.isEqual === "function")
+      {
+        code.push('function(a,b){return !!', clazz.classname, '.$$properties.',
+                  name, '.isEqual.call(this,a,b);};');
+      }
+      else if (typeof config.isEqual === "string")
+      {
+        var members = clazz.prototype;
+        // Name of member?
+        if (members[config.isEqual]!==undefined)
```

Comment:
members[config.isEqual] !== "undefined"
vs
typeof members[config.isEqual] !== "undefined"

The two do the same thing, but both are used in the same code, just a few lines apart, that's all.